src/wui/warehousewindow.cc

```diff
@@ -145,6 +146,25 @@ WarehouseWaresPanel::WarehouseWaresPanel(UI::Panel* parent,
 		ADD_POLICY_BUTTON(prefer, Prefer, _("Preferably store selected wares here"))
 		ADD_POLICY_BUTTON(dontstock, DontStock, _("Do not store selected wares here"))
 		ADD_POLICY_BUTTON(remove, Remove, _("Remove selected wares from here"))
+
+		buttons->add_inf_space();
+
+		if (upcast(InteractiveGameBase, igbase, &ib)) {
```

Comment:
Remove this check and let `EconomyOptionsWindow::create` accept an InteractiveBase. I'd just have to refactor it from IGBase to IBase for the scenario editor anyway ;)
Otherwise LGTM, not yet tested.

Reply:
Done